src/Data/Vec/Properties.agda

```diff
@@ -1099,6 +1099,20 @@ insert-punchIn xs       zero     v j       = refl
 insert-punchIn (x ∷ xs) (suc i)  v zero    = refl
 insert-punchIn (x ∷ xs) (suc i)  v (suc j) = insert-punchIn xs i v j
 
+-- where should this go? Or can I write toList-insert without this? Or does
+-- something similar already exist?
+length-length : (xs : Vec A n) → length xs ≡ List.length (toList xs)
+length-length []       = refl
+length-length (x ∷ xs) = cong suc (length-length xs)
```

Comment:
Not sure what exactly should I do with this 🙂

Reply:
I think this should be called to `length-toList` and should have the type `List.length (toList xs) ≡ length xs`